addons/web_editor/security/ir.model.access.csv

```diff
@@ -1,3 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_web_editor_converter_test,access_web_editor_converter_test,model_web_editor_converter_test,,1,1,1,1
 access_web_editor_converter_test_sub,access_web_editor_converter_test_sub,model_web_editor_converter_test_sub,,1,1,1,1
+access_web_editor_field_history_all,access.web.editor.field.history.all,model_field_html_history_diff,,0,0,0,0
+access_web_editor_field_history_portal,access.web.editor.field.history.portal,model_field_html_history_diff,base.group_portal,1,0,0,0
+access_web_editor_field_history_user,access.web.editor.field.history.user,model_field_html_history_diff,base.group_user,1,1,1,0
```

Comment:
Why would standard users be able to modify history ? I feel like this kind of technical storage should be admin-only, and accessed through dedicated controllers / methods that check related records access before entering a sudo mode to update it.

Idea is: if you are able to modify a document, you can modify its history (and same for read). Either you write extremely custom ACLs (see those in 'mail_message' :yum: ), either we make closed ACLs and use dedicated control methods or controllers. Especially you seem to access it through dediacted JS so quite easy to implement some control.

We generally do that for "side" content (like notifications, ratings, ...), maybe it deserves some discussion for that purpose :) .

Reply:
Thanks for all the idea and info:

related to https://github.com/odoo/odoo/pull/112957/files/6e396cf442dd0a227d8b6fe377eb7d88be0689d1#r1109739648